apps/astarte_appengine_api/test/astarte_appengine_api_web/controllers/groups_controller_test.exs

```diff
@@ -171,22 +173,25 @@ defmodule Astarte.AppEngine.APIWeb.GroupsControllerTest do
       assert json_response(create_again_conn, 409)["errors"] != nil
     end
 
-    test "creates the group with / in group name", %{conn: conn} do
-      group_name = "world/europe/italy"
-      encoded_group_name = "world%2Feurope%2Fitaly"
+    @tag issue: 904
+    property "creates the group with / in group name", %{conn: conn} do
+      check all group_name <- GroupTestGenerator.group_name(),
+                max_runs: 5 do
```

Comment:
if `max_runs` is hardcoded here, does it change according to `ci.exs` and `test.exs`?